index.html

```diff
@@ -4,231 +4,178 @@
+                <ul class="main-head flex items-center">
+                  <li class="subHeaderListStyle"><a href="about.html" class="no-underline gray-text-color">About</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Program</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Join</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">Sponsor</a></li>
+                  <li class="subHeaderListStyle"><a href="#" class="no-underline gray-text-color">News</a></li>
+                </ul>
+                <a href="#" class="compaign">Developer Campaign</a>
+              </nav>
+            </div>
+          </div>
+          <div class="head-container">
+            <div id="hamburger-icon">
+              <div class="bar1 bar"></div>
+              <div class="bar2 bar"></div>
+              <div class="bar3 bar"></div>
+            
+              <ul class="mobile-menu" id="mobile-menu">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#about">Program</a></li>
+                <li><a href="#contact">Join</a></li>
+                <li><a href="#about">Sponsor</a></li>
+                <li><a href="#contact">news</a></li>
+              </ul>
+            </div>
+            <div class="logo">
+              <a href="index.html"><img src="./images/logo1.png" class="logo-img" alt="developer summit logo"></a>
+            </div>
+          </div>
+        </header>
+  
+        <section class="home" id="home">
+          <div class="text-red">
+            <p class="hallo">"Hallo! Sharing World"</p>
+            <h1 class="h1">EAST AFRICAN DEVELOPER SUMMIT 2022</h1>
+          </div>
+         
+          <p class="intro">An opportunity for developpers, web designers, consultant, educator,
+             academics and businesses to share thought on how to advance web accessibility to everyone in East Africa
+          </p>
+          <h2 class="rv">20.07.2022(WED) ~ 12(FRI)</h2>
+          <p class="place">@ Palais des art, Boulevard de l'UPRONA, BUJUMBURA-BURUNDI</p>
+        </section>
       </div>
+
     </div>
-  </header>
-    <!-- end of mobile home section -->
+    
+    <section class="program-container">
+      <div class="head">
+        <p class="main1">Main Program</p>
+        <div class="red-rule"></div>
+      </div>
+      <div class="summit-program">
+        <div class="program-card">
+          <div class="card-title">
+            <i class="fa-solid fa-person-chalkboard fa-4x program-img"></i>
+            <p class="title">Speaches</p>
+          </div>
+          <p class="Opening">Listen to the opening speak of software specialists from different East African Countries giving 
+            key notes about IT in their respective countries.
+          </p>
```

Comment:
- [ ] At the moment, there is no proper padding in the program section. Kindly include padding to make it match the original design.
- [ ] Please also ensure that all cards are the same size

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169177895-9232f548-da60-419e-85b9-fdd41874a581.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169177851-163904e4-9b21-45c1-b44b-72da746eabc3.png)